lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -366,37 +366,58 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
   defp get_cause(:unknown), do: nil
   defp get_cause(cause), do: cause
 
+  # Special case for CR Departures; only supported on duo screens.
   defp placement(%__MODULE__{
          is_priority: true,
          screen: %Screen{
            app_params: %PreFare{
              cr_departures: %CRDepartures{
                enabled: true,
                pair_with_alert_widget: true
-             }
+             },
+             template: :duo
            }
          }
        }),
        do: :single_screen
 
   defp placement(
+         %__MODULE__{
+           is_terminal_station: is_terminal_station,
+           screen: %Screen{app_params: %PreFare{template: template}}
+         } = t
+       ) do
+    location = LocalizedAlert.location(t, is_terminal_station)
+    t |> duo_placement(location) |> adjust_placement(location, template)
+  end
+
+  defp duo_placement(
          %__MODULE__{
            alert: %Alert{effect: effect} = alert,
            is_priority: true,
            is_terminal_station: is_terminal_station,
            partial_closure_platform_names: []
-         } = t
+         } = t,
+         location
        )
        when effect in [:station_closure, :suspension, :shuttle] do
-    if LocalizedAlert.location(t, is_terminal_station) == :inside and
+    if location == :inside and
          LocalizedAlert.informs_all_active_routes_at_home_stop?(t) and
          (is_nil(Alert.direction_id(alert)) or is_terminal_station),
        do: :dual_screen,
        else: :single_screen
   end
 
-  defp placement(%__MODULE__{is_priority: true}), do: :single_screen
-  defp placement(%__MODULE__{}), do: :flex_zone
+  defp duo_placement(%__MODULE__{is_priority: true}, _location), do: :single_screen
+  defp duo_placement(%__MODULE__{}, _location), do: :flex_zone
+
+  # "Downgrade" placement by one level for solo screens. Exception: single-screen alerts that we
```

Comment:
Nit: delete we at end of comment so this makes sense

Reply:
Huh, I thought this did make sense... the "we" refers to the screen here (the screen is what's "inside" the alert). I can reword it since it was evidently confusing!

Reply:
ohhhhh I thought the screens was implied without the we. It's a weird thing to explain in a short comment lol